resources/definitions/zonestar_z6fb_printer.def.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "ZONESTAR Z6FB",
```

Comment:
I can't find any information about this printer. Can you provide a link to a spec sheet or purchasing page somewhere so that we can check this definition file?